tools/eksDistroBuildToolingOpsTools/pkg/consumerUpdater/bottlerocket.go

```diff
@@ -1,13 +1,24 @@
 package consumerUpdater
 
 import (
+	"bytes"
 	"fmt"
+	"github.com/aws/eks-distro-build-tooling/tools/eksDistroBuildToolingOpsTools/pkg/logger"
+	"os"
+	"path/filepath"
+	"regexp"
 
 	"github.com/aws/eks-distro-build-tooling/tools/eksDistroBuildToolingOpsTools/pkg/eksDistroRelease"
 )
 
 const (
-	bottlerocketName = "Bottlerocket"
+	bottlerocketName    = "Bottlerocket"
+	versionRegex        = `[0-9]+\.[0-9]+\.[0-9]+`
+	filePermission      = 0644
```

Comment:
For file permissions, do we want to give a name or description of the permissions given and moved to pkg/constants.go as well?

Reply:
done